### PR TITLE
Create 2022-11-10-fabric.md

### PR DESCRIPTION
**Pull request by darkfrog26**
Blog post to introduce Fabric

### CODE REVIEW

File: collections/_posts/2022-11-10-fabric.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+meta:
+nav: blog
+author: Matt Hicks
+---
+
+I know what you're thinking! "A new JSON library? Why? Don't we have plenty of those?" Well, the short answer is a
+resounding yes, but this library was born of the frustrations felt using those existing libraries. Don't get me wrong,
+there are some great JSON libraries in Scala and it's not my purpose to disparage them.  However, I assume others found
+the same frustrations I did with the existing libraries. This library benefited a great deal from the things I liked
+about the existing libraries and I went to great pains to make sure it didn't suffer from the same downsides they did.
```
Comment (jducoeur) [inline collections/_posts/2022-11-10-fabric.md:15]: Hmm.  While I don't think it's the end of the world, I would simply avoid the mild negative spin here.  I'd recommend instead just focusing on what Fabric excels at.  Which, yes, will somewhat duplicate the repo's README, but that's fine -- this is a push, and it's okay to say some of the same things that are in the fuller "pull" of reading the repo.

Comment (darkfrog26) [reply collections/_posts/2022-11-10-fabric.md:15]: @jducoeur I can appreciate that, but I'm not really sure to explain what Fabric excels at without contrasting it against the existing solutions. After all, if I don't have specific reasons not to use Circe or uPickle, why would you consider using my library?

Comment (darkfrog26) [reply collections/_posts/2022-11-10-fabric.md]: @jducoeur I drastically simplified the text. Hopefully, it's lost any negativity.

**Review by samspills — COMMENTED** [review]
Thank you for writing this up 🎉 

two technical blockers to address but otherwise the content of post sounds good to me

File: collections/_posts/2022-11-10-fabric.md (hunk 1 above)
Comment (jducoeur) [reply collections/_posts/2022-11-10-fabric.md]: Yep, the changed text works for me.  Thanks!

**Review by jducoeur — APPROVED** [review]
Sam's technical comments need to be dealt with, but I'm good to go once those are addressed.

**Comment by darkfrog26** [comment]
I suppose this needs to be merged first: https://github.com/typelevel/typelevel.github.com/pull/407

**Comment by darkfrog26** [comment]
@samspills thanks for the suggested changes, I appreciate it!

**Review by samspills — APPROVED** [review]
Thanks for the changes! I've approved the other PR as well :D

**Comment by darkfrog26** [comment]
Can someone with write access please merge both of these PRs?

**Comment by darkfrog26** [comment]
Thanks @jducoeur!

**Pull request merged**